easysid.cpp

```diff
@@ -14,10 +14,53 @@
 #endif
 
 SID sid;
-PcmOutput *pcm;
+PcmOutput *pcm = NULL;
+PcmOutput *wav = NULL;
+int sample_rate = 44100;
 
 extern "C" {
 
+EXPORT void start_capture(const char *fn)
+{
+    if (wav != NULL) {
+        // Deleting the existing object will flush, and close the current capture file.
+        delete wav;
+        wav = NULL;
+    }
+    wav = makePcmOutputWav(fn, sample_rate);
+}
+
+EXPORT void end_capture()
+{
+    if (wav != NULL) {
+        wav->flush();
```

Comment:
Why does this need an explicit call to `wav->flush()`, but no such call was necessary in the `start_capture` function? Make these consistent.